extended-utxo-spec/extended-utxo-specification.tex

```diff
@@ -942,36 +945,24 @@ \subsection{Validity of EUTXO-2 transactions}
       \]
     \item\textbf{Forging:}\\\\
       \begin{minipage}{0.85\textwidth}
-          \begin{enumerate}
-          \item
-              A transaction with a non-zero \forge{} field is only
-            valid if the ledger $\lambda$ is empty
-            (that is, if it is the initial
-            transaction). 
+        A transaction with a non-zero \forge{} field is only
+        valid if either:
+        \begin{enumerate}
+          \item the ledger $\lambda$ is empty (that is, if it is the initial transaction).
           \item \label{rule:custom-forge}
-            A transaction with a non-empty \customforge{} field is
-            only valid if for every key $h$ in $t.\customforge$, there
+            for every key $h$ in $\strictsupport(t.\forge)$, there
             exists $i \in t.\inputs$ and $(a,v,d) \in i.\outputs$ with
             $a =h$; in other words, some input must spend an output
             whose address is $h$.
-          \end{enumerate}
-          \end{minipage}
+        \end{enumerate}
+      \end{minipage}
     \item \textbf{Values are preserved}\\
       \begin{minipage}{0.85\textwidth}
-          \begin{enumerate}
-          \item\label{rule:native-value-is-preserved-2}
+          \label{rule:value-is-preserved-2}
             \[
-            t.\forge + \sum_{i \in t.\inputs} \getvalue(i, \lambda) = t.\fee + \sum_{o \in t.\outputs} o.\val
+            t.\forge + \sum_{i \in t.\inputs} \getvalue(i, \lambda) = \injectNative(t.\fee) + \sum_{o \in t.\outputs} o.\val
```

Comment:
`t.forge` is of type `QuantityMap+-` and `t.inputs` is of type `QuantityMap`, so technically we need to map the inputs into `QuantityMap+-` (doesn't help readability though)

Reply:
And then we also need to map the entire LHS back into `QuantityMap`. This ensures that all negative values in the `forge` field are matched by the transaction's inputs

Reply:
Sounds like another reason to do https://github.com/input-output-hk/plutus/issues/1515... I'll add a note like the one about N and Z saying we treat one as as subtype of the other.